lib/stat_tracker.rb

```diff
@@ -156,5 +156,17 @@ def team_info(team_id)
     # require 'byebug'; byebug
     teams_hash[:teams_info][team_id]
   end
+
+  def head_to_head(team_id)
```

Comment:
great job using the same helper method for these 3 methods

Reply:
🙏